### PR TITLE
Update Dockerfile, default_config.ini and bitsharesentry.sh

### PR DESCRIPTION
**Pull request by tperalta82**
Update Dockerfile to use ubuntu:bionic instead of phusion baseimage, had libcurl issues (have the issue on telegram)

Updated bitsharesentry.sh to add support to the BITSHARESD_ES_START_AFTER_BLOCK env var, to tell the witness_node to only start throwing data onto ES after the specified block

Updated default_config as per test-3.2.0 release

### CODE REVIEW

File: docker/default_config.ini [hunk 1]
```diff
@@ -35,7 +35,7 @@ rpc-endpoint = 0.0.0.0:8090
 enable-stale-production = false
 
 # Percent of witnesses (0-99) that must be participating in order to produce blocks
-required-participation = false
+required-participation = 0
```
Comment (oxarbitrage) [inline docker/default_config.ini]: The default parameter is `33`. We should change to that here.

**Comment by oxarbitrage** [comment]
Thank you for sending. I cant prove the baseimage thing but according to telegram discussions it seems to fix the curl issue. I sent a small change of the default participation value, rest looks good.

File: docker/default_config.ini (hunk 1 above)
Comment (tperalta82) [reply docker/default_config.ini]: Sure, will do !! Thanks

Comment (tperalta82) [reply docker/default_config.ini]: Please re-validate! Thanks a lot

**Comment by oxarbitrage** [comment]
Thanks, i am unsure if we will include the elasticsearch in the `release` branch so i am waiting to other opinions from the core team before merging, most probably tomorrow.

File: docker/default_config.ini [hunk 2]
```diff
@@ -35,7 +35,7 @@ rpc-endpoint = 0.0.0.0:8090
 enable-stale-production = false
 
 # Percent of witnesses (0-99) that must be participating in order to produce blocks
-required-participation = false
+required-participation = 33
```
Comment (pmconrad) [inline docker/default_config.ini]: Please comment it out instead. There's really no reason to set it explicitly.

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM phusion/baseimage:0.10.1
+FROM ubuntu:bionic
```
Comment (pmconrad) [inline Dockerfile]: We're not using stock ubuntu images for a reason. See https://phusion.github.io/baseimage-docker/ for explanations.
phusion/baseimage:0.11 is based on Ubuntu-18.04 - did you have a specific reason to move away from phusion?

**Comment by pmconrad** [comment]
Resolves #1823 #1824

File: Dockerfile (hunk 3 above)
Comment (oxarbitrage) [reply Dockerfile]: Agree on keeping phusion by changing to `0.11` if this gets the curl issue fixed. Thanks for the explications @pmconrad and sorry for not noticing.

Comment (tiago-dynx) [reply Dockerfile]: Will rebuild using phusion 0.11, if everything goes fine and starts with no issues, i'll push the changes, thanks a lot for the input guys

Comment (tperalta82) [reply Dockerfile]: Will rebuild using phusion 0.11, if everything goes fine and starts with no issues, i'll push the changes, thanks a lot for the input guys

File: docker/default_config.ini (hunk 2 above)
Comment (tperalta82) [reply docker/default_config.ini]: Will do

File: Dockerfile (hunk 3 above)
Comment (tperalta82) [reply Dockerfile]: Builds/Runs succesfully on 0.11, pushed code

File: docker/default_config.ini (hunk 2 above)
Comment (tperalta82) [reply docker/default_config.ini]: Done

**Comment by pmconrad** [comment]
Excellent, thanks!

**Pull request merged**